contracts/Account.cairo

```diff
@@ -75,12 +80,13 @@ func get_nonce{
 end
 
 @view
-func is_account{
-        syscall_ptr : felt*, 
-        pedersen_ptr : HashBuiltin*,
+func supportsInterface{
+        syscall_ptr: felt*, 
+        pedersen_ptr: HashBuiltin*, 
         range_check_ptr
-    }() -> (res: felt):
-    return (1)
+    } (interface_id: felt) -> (success: felt):
```

Comment:
```suggestion
    } (interfaceId: felt) -> (success: felt):
```
should we camelCase this? I don't remember what we're using

Reply:
I was under the impression we were just camelCasing the exposed methods' params of the token contracts. I could very well be mistaken. You thinking we should snake_case library params and camelCase contract params?

Reply:
It was more of an EIP compatibility thing, we're just camelCasing public interfaces, and this is part of it. Now that you mention it, I notice `IERC165` uses camelcase for the fn name but snakecase for the parameter.